code/player/TTTPlayer.AFK.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Sandbox;
+
+using TTTReborn.Settings;
+
+namespace TTTReborn.Player
+{
+    public partial class TTTPlayer
+    {
+        public readonly static List<InputButton> Buttons = Enum.GetValues(typeof(InputButton)).Cast<InputButton>().ToList();
+        private static int SecondsTillKick => ServerSettings.Instance.AFK.MinutesTillKick * 60;
```

Comment:
`SecondsTillKick` and then the server setting is `MinutesTillKick`, we should pick one. I think typically in the codebase we define everything in `Seconds`.

Reply:
Agreed. Will make this change.